dynamic-layers/raspberrypi/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -3,9 +3,12 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 SRC_URI += "\
     file://add-reset-info.patch \
     file://enable-reset-info-cmd-fragment.cfg \
+    file://fix-usb-scanning-upstream.patch \
+    file://fix-usb-scanning-balena.patch \
     file://omnect_env.patch \
     file://rpi-always-set-fdt_addr-with-firmware-provided-FDT-address.patch \
     file://omnect_env_rpi.h \
+    file://0001-usb-xhci-ring.c-don-t-queue-URB-for-halted-endpoint.patch \
```

Comment:
why do we need this patches?
how do we plan on getting rid of them?
does upstream plan to integrate them?
please add comments.

can we add them depending on distro feature "3g"?

Reply:
we deciced to deactivate usb for now